.gitignore

```diff
@@ -1 +1 @@
-node_modules/
\ No newline at end of file
+node_modules/
```

Comment:
Please reset this file. It seems there used to be a new line at the end of the file which has been removed.